R/R.r

```diff
@@ -50,10 +51,24 @@ r_env_vars <- function() {
     "TAR" = auto_tar())
 
   if(is.na(Sys.getenv("NOT_CRAN", unset = NA))) {
-    c(vars, "NOT_CRAN" = "true")
-  } else {
-    vars
+    vars <- c(vars, "NOT_CRAN" = "true")
```

Comment:
It just occurred to me that this would be nicer as `vars[["NOT_CRAN"]] <- "true"`